gradle-baseline-typescript/build.gradle

```diff
@@ -57,6 +57,10 @@ gradlePlugin {
     }
 }
 
+tasks.named('processResources').configure {
+    duplicatesStrategy 'include'
```

Comment:
do you know where the conflicts are coming from?

Reply:
this is coming from the resource we explicitly add (in the resource directory) and one generated by the gradle-plugin plugin. We need the explicit resource for testing to work and the plugin also manages publishing